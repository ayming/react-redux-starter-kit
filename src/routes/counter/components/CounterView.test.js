/* eslint-env jest */
import React from 'react'
import renderer from 'react-test-renderer'

import CounterView from './CounterView'
import noop from '../../../utils/noop'

describe('CounterView', () => {
  test('renders correctly', () => {
    const wrapper = renderer
      .create(<CounterView counter={1} increment={noop} decrement={noop} />)
      .toJSON()

    expect(wrapper).toMatchSnapshot()
  })
})
