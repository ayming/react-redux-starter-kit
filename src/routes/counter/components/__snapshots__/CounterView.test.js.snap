// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CounterView renders correctly 1`] = `
<div
  className="wrapper"
>
  <p>
    Counter: 
    1
     
    <button
      onClick={[Function]}
      type="button"
    >
      +
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      -
    </button>
  </p>
  <a
    href="/"
    method="push"
    onClick={[Function]}
  >
    HOME
  </a>
   
  <a
    href="/about"
    method="push"
    onClick={[Function]}
  >
    ABOUT
  </a>
</div>
`;
